package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "metal-a11y-checker",
-  "version": "1.0.2",
+  "version": "1.0.4",
```

Comment:
This is going to conflict with the existing `1.0.3` version. Please, rebase first.